pontoon/base/migrations/0093_translationmemoryentry_project.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-05-09 21:26
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('base', '0092_locale_db_collation'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='translationmemoryentry',
+            name='project',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='base.Project'),
```

Comment:
Hm, this line seems to be out of sync with line https://github.com/mozilla/pontoon/pull/606/files#diff-c18ce63ddb8aca155c0553a0c1b174e3R2123?